aedes.js

```diff
@@ -80,7 +82,19 @@ function init () {
     heartbeatInterval: 500
   })
 
-  createServers(broker.handle)
+  if (DB.waitForReady) {
+    broker.persistence.once('ready', function () {
+      DB.cleanDb(broker.persistence, function (err) {
+        if (err) {
+          throw err
```

Comment:
don't do this. Also, it looks like `init()` should be async.

Reply:
What should I do with that error? 

> Also, it looks like init() should be async.

Why? Should I `await createServers`?

Reply:
This is an asynchronous operations with no callback, it should have one. This error should be collected and handled centrally (crashing the process is fine).

Reply:
Check it now